programs/eosioc/main.cpp

```diff
@@ -560,6 +560,7 @@ int main( int argc, char** argv ) {
    string table;
    string lower;
    string upper;
+   string key;
```

Comment:
This appears to go completely unused.

Reply:
already used!! 
eg.
eosc      get table oct oct account
eosc      get table songbo oct account
eosc      get table zhaobo oct account
when account table rows large enough, limit become import !!!